script.js

```diff
@@ -0,0 +1,41 @@
+
+var userName = prompt("Hi there. What is your name?");
+
+if (userName!= null) {
+   alert("Welcome " + userName + ".  My name is Charlie.  I am a program that will assess potential personality traits and how you may learn best.  Let's get started. ");
+}
+
+//First Selection (Choice of Profession)
+
+var jobChoices = ["lawyer" , "president" , "garbage man" , "chef", "entrepreneur"];
+var userJob =  prompt("What do you want to be? A " + jobChoices );
+
```

Comment:
good leveraging of an array. there are ways to render the contents of the array in a more readable fashion. you could write a for-loop that takes elements of the array and builds a string with spaces:

```
var stringToBuild = "";
for (i =0; i < jobChoices.length; i++){
    stringToBuild += jobChoices[i] + " ";
}
```